.github/CODEOWNERS

```diff
@@ -34,3 +31,5 @@
 Cargo.lock @FuelLabs/leads
 /deployment/ @FuelLabs/leads
 
+# Everything else
+* @FuelLabs/leads
```

Comment:
This does the opposite of what you think.

CODEOWNER rules are set to the last matching rule for every changed file. putting the catchall at the end means every file has to be reviewed by a team lead, which is not what we want.

Reply:
Ah sorry, I thought that's what was happening, but you're right. I'll remove it.